tests/test_cli.py

```diff
@@ -20,6 +25,51 @@ def context():
     return ctx().obj["context"]
 
 
+mock_gui_main = """from .app import show
```

Comment:
I think he's referring to a Python *package* called `mock_gui`, and not a module (file). Are you sure you've tried calling just the module?